bigtable-hbase-parent/bigtable-hbase-shaded/pom.xml

```diff
@@ -27,14 +27,104 @@ limitations under the License.
     <packaging>jar</packaging>
     <name>${project.groupId}:${project.artifactId}</name>
     <description>
+       DEPRECATED: Please use bigtable-hbase-1.x or bigtable-hbase-1.x-hadoop.
```

Comment:
There should not be any users of bigtable-hbase-shaded.  Can you please delete the project if it's no longer referenced?